### DISCUSSION TITLE
Removed from HACS?

### DISCUSSION THREAD

**Discussion by benjinho87** [lkempf/casambi-bt-hass#71] [Q&A] [answered]
Hi, I wanted to re-install the integration, but I can't find it anymore on HACS since yesterday. Has it been removed? Only the "non-bluetooth" casambi integration shows up.

Another question: I needed to re-install the integaration as I did an update to  0.1.1. After the update, all scences did not work anymore with the error"object has no attribute 'scene'". It used to work on 0.1.0 perfectly. After downgrading to 0.1.0 (on this time the integration was still in HACS), scenes did work again, but the integration did only found my main casambi network as a device, but not all lights.

Thanks for support.

**Comment by lkempf** [comment, marked as answer]
You might have to add the repo in HACS again (no idea why though). Please open an issue an issue regarding your problem with scenes so that I can fix it (I need a bit more information for that)
